qoffer-infrastructure/src/main/resources/offer-template/offer.html

```diff
@@ -0,0 +1,195 @@
+            
+            <div class="small-spacer"></div>
+            
+            <div class="info-title">Offer valid until</div>
+            <div id="offer-expiry-date">30-06-2021</div>
+            
+            <div class="small-spacer"></div>
+            
+            <div class="info-title">Delivery time</div>
+            <div id="offer-expiry-date">Approx. 4-8 weeks upon data retrieval.</div>
+
+        </div>
+        <div class="project-section">
+            <div id="project-title">Project title</div>
+            <div class="project-content-topic">Objective</div>
+            <div id="objective" class="project-content">Describe what the project is about here. Make sure not to write in too much detail as we want to prevent unauthorized people to get too much knowledge about the project.
+            </div>
+
+            <div class="project-content-topic">Experimental design</div>
+            <div id="experimental-design" class="project-content">Explain the experimental design in detail.</div>
+
+            <div class="project-content-topic">Bioinformatics analysis</div>
+            <div id="analysis" class="project-content">Explain in short what analysis will be performed.</div>
+
+            <div id="quote-description" class="project-content">Describe what is included in the quotation/cost estimation and some general information.</div>
+
+            <div class="cost-summary">
+                <div class="">Estimated total (net)</div>
+                <div id ="total-costs">1,000.00 €</div>
+            </div>
+        </div>
+        <div class="signature">
+            <hr width="30%" align="left"/>
+            <p>Date, Signature</p>  
+    
+        </div>     
+    </div>
+    
+    <hr size="1" noshade/>
+    <footer>
+        <div class="grid-container-footer">
+            <div>
+               <strong>Quantitative Biology</br> Center (QBiC)</strong> </br>
+               Dr. Sven Nahnsen </br>
+               Auf der Morgenstelle 10 </br>
+               72076 Tübingen
+            </div>
+            <div>
+                <strong>Institute for Medical Genetics and Applied Genomics (IMGAG)</strong> </br>
+                Prof. Dr. Olaf Rieß </br>
+                Calwerstr. 7 </br>
+                72076 Tübingen
+             </div>
+            <div>
+                <strong>Institute for Medical Microbiology and Hygiene </br>(MGM)</strong> </br>
+                Prof. Dr. Julia-Stefanie Frick </br>
+                Elfriede-Aulhorn-Str. 6 </br>
+                72076 Tübingen
+            </div>
+            <div>
+                <strong>Proteome Center </br>Tübingen (PCT) </strong></br>
+                Prof. Dr. Boris Macek </br>
+                Auf der Morgenstelle 15 </br>
+                72076 Tübingen
+            </div>
+            <div>
+                <strong>Core Facility for Medical Bioanalytics (CFMB) </strong></br>
+                Prof. Dr. Marius Üffing </br>
```

Comment:
checked the document online, the encoding seems to be not correct. Umlaute will not be shown correctly and the € will also not be shown correctly